modules/dxp/apps/segments/segments-experiment-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -1,70 +1,75 @@
 @import 'atlas-variables';
+@import 'cadmin-variables';
 @import 'mixins';
 
 @import './click-goal';
 
-.lfr-segments-experiment-panel {
-	&.open-admin-panel.sidenav-menu-slider {
-		visibility: visible;
-		width: 320px;
-	}
+html#{$cadmin-selector} {
+	.cadmin {
+		&.lfr-segments-experiment-panel {
+			&.open-admin-panel.sidenav-menu-slider {
+				visibility: visible;
+				width: 320px;
+			}
 
-	.sidebar {
-		border-left: 1px solid $gray-300;
-	}
-}
+			.sidebar {
+				border-left: 1px solid $gray-300;
+			}
+		}
 
-.lfr-has-segments-experiment-panel {
-	overflow: auto;
+		.lfr-has-segments-experiment-panel {
+			overflow: auto;
 
-	.lfr-configurator-visibility {
-		display: none;
-	}
-}
-
-.lfr-segments-experiment-panel {
-	.btn-secondary {
-		svg.lexicon-icon-ellipsis-v {
-			fill: #6b6c7e;
+			.lfr-configurator-visibility {
+				display: none;
+			}
 		}
-	}
 
-	.sidebar .sidebar-header {
-		padding-bottom: 0.75rem;
-		padding-top: 0.75rem;
-	}
-	@media only screen and (min-width: 768px) {
-		.sidebar-header {
-			padding-bottom: 12px;
-			padding-left: 24px;
-			padding-top: 12px;
-		}
-	}
-	.sidebar-header {
-		height: 64px;
-		line-height: 32px;
+		&.lfr-segments-experiment-panel {
+			.btn-secondary {
+				svg.lexicon-icon-ellipsis-v {
+					fill: #6b6c7e;
+				}
+			}
 
-		.lexicon-icon {
-			font-size: 14px;
-		}
-	}
-	.sidebar-body {
-		bottom: 0;
-		left: 0;
-		padding: 0;
-		position: absolute;
-		right: 0;
-		top: 56px;
-	}
-	@media only screen and (min-width: 768px) {
-		.sidebar-header {
-			padding-bottom: 12px;
-			padding-left: 24px;
-			padding-top: 12px;
-		}
-	}
+			.sidebar .sidebar-header {
+				padding-bottom: 0.75rem;
+				padding-top: 0.75rem;
+			}
+			@media only screen and (min-width: 768px) {
+				.sidebar-header {
+					padding-bottom: 12px;
+					padding-left: 24px;
+					padding-top: 12px;
+				}
+			}
+			.sidebar-header {
+				height: 64px;
+				line-height: 32px;
 
-	dt {
-		font-weight: 500;
+				.lexicon-icon {
+					font-size: 14px;
+				}
+			}
+			.sidebar-body {
+				bottom: 0;
+				left: 0;
+				padding: 0;
+				position: absolute;
+				right: 0;
+				top: 56px;
+			}
+			@media only screen and (min-width: 768px) {
```

Comment:
Should use media query mixin here `@include media-breakpoint-up(md, $cadmin-grid-breakpoints) {`